app/src/main/java/com/x8bit/bitwarden/data/autofill/accessibility/BitwardenAccessibilityService.kt

```diff
@@ -22,7 +22,6 @@ class BitwardenAccessibilityService : AccessibilityService() {
     lateinit var processor: BitwardenAccessibilityProcessor
 
     override fun onAccessibilityEvent(event: AccessibilityEvent) {
-        if (rootInActiveWindow?.packageName != event.packageName) return
```

Comment:
Posting discussion results for posterity. This is in place as a security measure. We will try deferring the call until other prerequisites are met to _hopefully_ reduce the performance impact.